### ISSUE TITLE
Change the example in the documentation of Transpose

### ISSUE THREAD

**Issue by agrastien** [onnx/onnx#6329]
TL;DR: 

The ONNX documentation for operator `Transpose` does not remove an ambiguity.  The current example is 

> For example, when perm=(1, 0, 2), given an input tensor of shape (1, 2, 3), the output shape will be (2, 1, 3).

I propose to replace it with

> For example, when perm=(2, 0, 1), given an input tensor of shape (1, 2, 3), the output shape will be (3, 1, 2).

Long version:

Applying the permutation `perm` over a tensor of shape `(d1,...,dk)` leads to a tensor of shape `(d'1,...,d'k)` such that `d'i = d(perm[i])`.  Another (slightly less natural but reasonable) interpretation would be to assume instead that `d'(perm[i]) = di`. (Indeed, that's the inverse permutation of `perm`).  The current example from the documentation is such that it is its own inverse, which means that it does not disambiguate the two interpretations.  In the example I propose, the incorrect interpretation would lead to a tensor of shape (2,3,1).  Hence, the example is more informative.

**Comment by gramalingam** [comment]
Good point. https://github.com/onnx/onnx/pull/6072 was created for exactly the same reason, but it was abandoned because many unrelated files got included in the PR :-( ... would be good to recreate it.

**Comment by scha54** [comment]
Is this issue being worked upon by someone?

**Comment by Br1an67** [comment]
I'd like to work on this. Replace the first example perm=(1, 0, 2) with perm=(2, 0, 1) to disambiguate the two possible interpretations.

PR: #7701
